.github/workflows/integration-x86-64.yaml

```diff
@@ -30,12 +33,8 @@ jobs:
       - name: Load openvswitch module
         run: sudo modprobe openvswitch
       - name: Run integration tests
-        run: scripts/dev_cli.sh tests --integration
+        timeout-minutes: 20
+        run: scripts/dev_cli.sh tests --integration --libc ${{ matrix.libc }}
```

Comment:
I just noticed that the step `Run unit tests` is always running with gnu, say it needs to be updated to `scripts/dev_cli.sh tests --unit --libc ${{ matrix.libc }}`